st2api/st2api/cmd/api.py

```diff
@@ -64,10 +65,25 @@ def _run_server():
     host = cfg.CONF.api.host
     port = cfg.CONF.api.port
 
-    LOG.info('(PID=%s) ST2 API is serving on http://%s:%s.', os.getpid(), host, port)
+    api_host = cfg.CONF.api.host
+    api_port = cfg.CONF.api.port
 
-    wsgi.server(eventlet.listen((host, port)), app.setup_app())
-    return 0
+    webui_host = api_host
+    webui_port = int(api_port) + 1
+
+    LOG.info('(PID=%s) ST2 API is serving on http://%s:%s.', os.getpid(), api_host, api_port)
+    LOG.info('(PID=%s) ST2 WebUI is serving on http://%s:%s.', os.getpid(), webui_host, webui_port)
+
+    def _run_api_server():
+        wsgi.server(eventlet.listen((api_host, api_port)), get_api_app())
+
+    def _run_webui_server():
```

Comment:
This approach is not ideal, but pecan doesn't allow multiple static_roots, so that's a compromise I had to make. I could make it to work by re-using the same app, but this means I would need to set `debug=True` for the actual API server.

To make it easier to deploy and run, I just run it as part of the st2api process on port api_port + 1. Additional process and configuration file just makes deployment more complicated which is unnecessary.

And to make it clear (we should point this out in the docs) this server is there only for convenience - for actual production deployments, people should serve static files using nginx / apache.